ugs-core/src/com/willwinder/universalgcodesender/TinyGController.java

```diff
@@ -292,11 +293,7 @@ public void viewParserState() throws Exception {
 
     @Override
     public void updateParserModalState(GcodeCommand command) {
-        if (command.getCommandString().startsWith("{\"gc")) {
-            String gcode = StringUtils.substringBetween(command.getCommandString(), "{\"gc\":\"", "\"");
-            GcodeCommand gcodeCommand = new GcodeCommand(gcode);
-            super.updateParserModalState(gcodeCommand);
-        }
+        // This is handled by the rawResponseHandler
```

Comment:
This calls `parser.addCommand` which updates the on-the-fly GcodeParser, that may need to be called somewhere. I think my plan was to deprecate distanceModeCode / unitsCode in favor of the real gcode parser.